Stratis.Bitcoin/Logging/Logs.cs

```diff
@@ -116,74 +88,18 @@ public static ILoggerFactory GetLoggerFactory(string[] args)
             //var logipsArgs = args.GetValueOf("-logips");
             //var printtoconsoleArgs = args.GetValueOf("-printtoconsole");
 
-            ILoggerFactory loggerFactory = new LoggerFactory()
-                .WithFilter(filterSettings);
-            loggerFactory.AddDebug(LogLevel.Trace);
-            loggerFactory.AddConsole(LogLevel.Trace);
-            loggerFactory.AddFile("Logs/node-{Date}.json", isJson: true, minimumLevel: LogLevel.Trace, fileSizeLimitBytes: 5000000);
-            return loggerFactory;
-        }
-
-        public static ILogger Configuration
-        {
-            get; private set;
+            loggerFactory.WithFilter(filterSettings);
         }
 
-        public static ILogger RPC
+        public static void AddPath(this ILoggerFactory loggerFactory, DataFolder dataFolder)
```

Comment:
this method should probably be called AddFile and just be a wrapper for the AddFile method, with the parameters defined in the caller .